app/controllers/openid_connect/logout_controller.rb

```diff
@@ -3,7 +3,6 @@ class LogoutController < ApplicationController
     include SecureHeadersConcern
     include FullyAuthenticatable
 
-    before_action :apply_secure_headers_override, only: [:index, :delete]
```

Comment:
Did we not need it on `:index` ? I see we're only applying the new logic on `:delete`.

Reply:
Oh, I missed that it is called indirectly still, via `handle_successful_logout_request`